src/kili/use_cases/organization/use_cases.py

```diff
@@ -0,0 +1,96 @@
+"""Organization use cases."""
+
+from dataclasses import dataclass
+from typing import Dict, Generator, Optional
+
+from kili.adapters.kili_api_gateway.helpers.queries import QueryOptions
+from kili.adapters.kili_api_gateway.organization.types import (
+    KiliAPIGateWayCreateOrganizationInput,
+    KiliAPIGateWayUpdateOrganizationInput,
+)
+from kili.domain.organization import (
+    Organization,
+    OrganizationFilters,
+    OrganizationId,
+    OrganizationMetricsFilters,
+)
+from kili.domain.types import ListOrTuple
+from kili.use_cases.base import BaseUseCases
+
+
+@dataclass
+class OrganizationToCreateUseCaseInput:
+    """Organization to create use case input."""
+
+    name: str
+    address: str
+    city: str
+    country: str
+    zip_code: str
```

Comment:
kw_only

Reply:
available only from Python 3.10

Reply:
oh ok I didn't know

that's one of the advantages of using attrs, dataclasses features are tied to the python version :/